.github/workflows/openms_ci_matrix_full.yml

```
@@ -408,8 +408,15 @@ jobs:
             export ENABLE_CWL=ON
           fi
 
+          # Install Python build and test dependencies from pyproject.toml for pyOpenMS CMake configure
           if [[ "${PYOPENMS:-}" == "ON" ]]; then
-            python -m pip install -r $GITHUB_WORKSPACE/OpenMS/src/pyOpenMS/requirements_bld.txt
+            pip install -q $(python3 -c "
+import tomllib
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)